src/Twig.php

```diff
@@ -6,10 +7,12 @@
  * @copyright Copyright (c) 2011-2015 Josh Lockhart
  * @license   https://github.com/slimphp/Twig-View/blob/master/LICENSE.md (MIT License)
  */
+
 namespace Slim\Views;
 
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
```

Comment:
This component deliberately doesn't couple to Pimple so that it can be used with other DICs.